conda/models/package_info.py

```diff
@@ -2,9 +2,13 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
We should have a `tests/models/test_package_info.py` file that is just a unit test or two that loads some contrived data.